src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -90,7 +91,7 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
       <PageHeader.Left>
         <Display.Desktop>
           <Flex alignItems='center' mr='auto' flex={1}>
-            <AssetIcon assetId={asset.assetId} boxSize='40px' />
+            <AssetIcon assetId={asset.assetId} pairProps={pairProps} />
```

Comment:
Used in:

- `<AssetAccountDetails />`
- `<AccountDetails />`
- `<AccountTokenTxHistory />`

Give me some testing love in all 3

Reply:
✅